### PR TITLE
[zh-cn] Resync tasks/

### PR DESCRIPTION
**Pull request by mengjiao-liu**
### Description

Synchronize the following files:
```
content/zh-cn/docs/tasks/configure-pod-container/ss-readiness-startup-probes.md
content/zh-cn/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy.md
content/zh-cn/docs/tasks/administer-cluster/kubeadm/change-package-repository.md
content/zh-cn/docs/tasks/administer-cluster/kubeadm/upgrading-linux-nodes.md
content/zh-cn/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md
content/zh-cn/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md
content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md
content/zh-cn/docs/tasks/extend-kubectl/kubectl-plugins.md
```

### CODE REVIEW

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -41,20 +41,25 @@ is observed as not-ready for some period of time before it is hard killed.
...
 it succeeds, making sure those probes don't interfere with the application startup.
 This can be used to adopt liveness checks on slow starting containers, avoiding them
 getting killed by the kubelet before they are up and running.
 -->
-kubelet 使用就绪探针可以知道容器何时准备好接受请求流量，当一个 Pod
-内的所有容器都就绪时，才能认为该 Pod 就绪。
+kubelet 使用就绪探针可以知道容器何时准备好接受请求流量。
 这种信号的一个用途就是控制哪个 Pod 作为 Service 的后端。
-若 Pod 尚未就绪，会被从 Service 的负载均衡器中剔除。
+当 Pod 的 `Ready` [条件](/zh-cn/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions) 为
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:59]:
```suggestion
当 Pod 的 `Ready` [状况](/zh-cn/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions) 为
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 2]
```diff
@@ -41,20 +41,25 @@ is observed as not-ready for some period of time before it is hard killed.
...
+true 时，Pod 被认为是就绪的。若 Pod 尚未就绪，会被从 Service 的负载均衡器中剔除。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:60]:
```suggestion
true 时，Pod 被认为是就绪的。若 Pod 未就绪，会被从 Service 的负载均衡器中剔除。
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 3]
```diff
@@ -41,20 +41,25 @@ is observed as not-ready for some period of time before it is hard killed.
...
+当 Pod 的节点的 `Ready` 条件不为 true 时、当 Pod 的某个 `readinessGates` 为 false
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:61]:
```suggestion
当 Pod 所在节点的 `Ready` 状况不为 true 时、当 Pod 的某个 `readinessGates` 为 false
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 4]
```diff
@@ -41,20 +41,25 @@ is observed as not-ready for some period of time before it is hard killed.
...
+时，或者当 Pod 中有任何一个容器未就绪时，Pod 的 `Ready` 条件为 false。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:62]:
```suggestion
时，或者当 Pod 中有任何一个容器未就绪时，Pod 的 `Ready` 状况为 false。
```

**Comment by mengjiao-liu** [comment]
All comments addressed.

**Review by windsonsea — COMMENTED** [review]
/approve

**Pull request merged**